products/fuse/get-started.adoc

```diff
@@ -4,10 +4,6 @@
 == Prerequisites section title
 Prerequisites &#38; Required Dependencies Checklist
 
-== Prerequisites section
-1. Java SE Development Kit (JDK), we recommend OpenJDK or Oracle JDK.
```

Comment:
@paulrobinson @catrobson - I think I misunderstood the comments in the JIRA ticket.  I will fix the PR.